src/Miunie.Discord/Convertors/MiunieChannelConvertor.cs

```diff
@@ -0,0 +1,43 @@
+using Miunie.Core;
+using DSharpPlus.Entities;
+using DSharpPlus.CommandsNext.Converters;
+using DSharpPlus.CommandsNext;
+using System.Threading.Tasks;
+using System;
+
+namespace Miunie.Discord.Convertors
+{
+    public class MiunieChannelConvertor : IArgumentConverter<MiunieChannel>
+    {
+        private readonly DiscordChannelConverter _dcConverter;
+
+        public MiunieChannelConvertor()
+        {
+            _dcConverter = new DiscordChannelConverter();
+        }
+        public async Task<Optional<MiunieChannel>> ConvertAsync(string userInput, CommandContext context)
+        {
+            var result = await _dcConverter.ConvertAsync(userInput, context);
+            return DiscordChannelToMiunieChannel(result.Value);
+        }
+
+        public MiunieChannel DiscordChannelToMiunieChannel(
+                                                        DiscordChannel channel)
```

Comment:
Code smell, you wrap because the signature is unnecessarily long. Think about the context
`MiunieChannelConvertor->DiscordChannelToMiunieChannel`
3x Channel combo :zap: 
`MiunieChannelConvertor->FromDiscordChannel`
conveys the same information given the context.